src/main/java/org/folio/rest/impl/EholdingsProvidersImpl.java

```diff
@@ -53,12 +55,11 @@ public EholdingsProvidersImpl(RMAPIConfigurationService configurationService,
 
   @Override
   @Validate
+  @HandleValidationErrors
   public void getEholdingsProviders(String q, String sort, int page, int count, Map<String, String> okapiHeaders, Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
-    if (!headerValidator.validate(okapiHeaders, asyncResultHandler)) {
-      return;
-    }
+    headerValidator.validate(okapiHeaders);
     if(sort != null && !Sort.contains(sort.toUpperCase())){
-      throw new ValidationException("sort parameter must have value \"relevance\" or \"name\"");
+      throw new ValidationException("Invalid sort parameter");
     }
     Sort nameSort = sort != null ? Sort.valueOf(sort.toUpperCase()) : null;
     CompletableFuture.completedFuture(null)
```

Comment:
If sort is null -- we should apply logic to set it base on whether or not query string q has a value. Below links in ruby code show this logic for providers and titles both. If query string is null -- sort should default to relevance

https://github.com/folio-org/mod-kb-ebsco/blob/f4a8c7e6a63373a24cb78ff277c7b8e65492a2c3/app/models/title.rb#L61

https://github.com/folio-org/mod-kb-ebsco/blob/f4a8c7e6a63373a24cb78ff277c7b8e65492a2c3/app/models/provider.rb#L21